app/src/features/apiClient/slices/entities/api-client-variables.ts

```
@@ -0,0 +1,56 @@
+      ...variableData,
+    };
+    this.unsafePatch(s => {
+      const variables = this.getVariableObject(s);
+      variables[key] = variable;
+    });
+  }
+
+  set(params: Pick<Variable, 'id'> & Partial<Omit<Variable, 'id'>>) {
+    const {id, ...variableData} = params;
+    this.unsafePatch(s => {
+      debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)